openreview/conference/builder.py

```diff
@@ -1925,10 +1925,10 @@ def get_invitees(self, conference, number):
 
 class MetaReviewStage(object):
 
-    def __init__(self, start_date = None, due_date = None, public = False, additional_fields = {}, remove_fields=[], process = None):
+    def __init__(self, start_date = None, due_date = None, public = False, additional_fields = {}, remove_fields=[], process = None, name='Meta_Review'):
```

Comment:
```suggestion
    def __init__(self, name='Meta_Review', start_date = None, due_date = None, public = False, additional_fields = {}, remove_fields=[], process = None):
```

Reply:
I tried it like this, but some tests don't pass with this change. I'm not sure why.